warehouse/accumulo-extensions/src/test/java/datawave/ingest/table/balancer/ShardedTableTabletBalancerTest.java

```diff
@@ -257,7 +258,7 @@ public void testNoRebalanceWithPendingMigrationsForServer() {
                 new TabletIdImpl(new KeyExtent(bar, new Text("2"), new Text("1"))),
                 new TabletIdImpl(new KeyExtent(TNAME, new Text("2"), new Text("1"))));
         //@formatter:on
-        long balanceWaitTime = testBalancer.balance(new BalanceParamsImpl(testTServers.getCurrent(), migrations, migrationsOut));
+        long balanceWaitTime = testBalancer.balance(new BalanceParamsImpl(testTServers.getCurrent(), migrations, migrationsOut, Ample.DataLevel.USER));
```

Comment:
Accumulo PR 5195 would break this when merged as it is now.  Need to get 5195 merged before 2.1.4 release as it fixes a few balancing bugs.  The bugs it fixes are referenced in a later comment on the PR, I wrote an IT after opening the PR and found a few bugs from running that IT.

Reply:
Okay, that's good to know. I'll add a blocker label to #5195

Reply:
@ddanielr so should this pr be blocked on waiting for 5195 to be merged and this getting updated?

Reply:
5195 is merged. 
Will address breaking changes as they pop up in the build